src/api/spec/lib/feature_switch/feature_spec.rb

```diff
@@ -1,22 +1,25 @@
 require 'rails_helper'
+require 'yaml'
 
 RSpec.describe Feature do
-  let(:user) { create(:confirmed_user, :in_beta, login: 'Tom') }
-  subject { Feature.active?('bootstrap') }
+  subject { Feature.active?(:bootstrap) }
 
   context 'with beta users' do
-    before do
-      User.session = user
-      Feature.instance_variable_set(:@perform_initial_refresh_for_user, true)
+    let(:beta) do
+      Feature.use_beta_features(true)
+      Feature.refresh!
```

Comment:
why assign it with let if you don't need it? Please add it to the ```before``` block

Reply:
I looked for rspec guidance and it was preffered over functions - because that's what it is, a function that i need in 2 before blocks

Reply:
not sure where you are searching for rspec guidance. The better specs say:

"When you have to assign a variable instead of using a before block to create an instance variable, use let. Using let the variable lazy loads only when it is used the first time in the test and get cached until that specific test is finished. "  

which value are you assiging to beta? The method calls belong to ```before``` block

Reply:
they are in 2 before blocks and so I needed a function - if you put them in the context's before block they are run in the wrong order. https://stackoverflow.com/a/17778760/11191489 - but I'll use a method to unconfuse you

Reply:
I'm not confused, I just want clean code. Please don't mark the reviews as resolved. If they are still not.